spanner/spansql/parser.go

```diff
@@ -1495,6 +1542,98 @@ func (p *parser) parseColumnOptions() (ColumnOptions, *parseError) {
 	return co, nil
 }
 
+func (p *parser) parseDatabaseOptions() (DatabaseOptions, *parseError) {
+	debugf("parseDatabaseOptions: %v", p)
+	/*
+			options_def:
+				OPTIONS (enable_key_visualizer = { true | null },
+		                 optimizer_version = { 1 ... 2 | null },
+		                 version_retention_period = { 'duration' | null })
+	*/
+
+	if err := p.expect("OPTIONS"); err != nil {
+		return DatabaseOptions{}, err
+	}
+	if err := p.expect("("); err != nil {
+		return DatabaseOptions{}, err
+	}
+
+	// We ignore case for the key (because it is easier) but not the value.
+	var opts DatabaseOptions
+	for {
+		if p.eat("enable_key_visualizer", "=") {
+			tok := p.next()
+			if tok.err != nil {
+				return DatabaseOptions{}, tok.err
+			}
+			enableKeyVisualizer := new(bool)
+			switch tok.value {
```

Comment:
if the case is `false`, what would happen here?

Reply:
It will fall through to the default statement, which indicates that it is an invalid value, so it will return an error. The documentation currently does not include this option, so I'm actually not 100% sure what the valid values are, but [looking at the `allow_commit_timestamp` boolean option for columns](https://cloud.google.com/spanner/docs/data-definition-language#alter_table) it seems that boolean options (with `false/null` as the default value) only accept `true` or `null` as values.